_includes/sidebar.html

```diff
@@ -89,32 +93,38 @@
                     return "stable";
                   })();
 
-                  function renderItems(items, tier) {
+                  function renderItems(items, paths) {
                     if (!items || items.length == 0)
                         return $();
 
                     var lis = items.map(function (item) {
                       var urls = (item.urls || []).map(function (url) {
                           return baseUrl + url.replace("${VERSION}", pageVersion);
                       });
-                      var subitems = renderItems(item.items, tier + 1);
+                      var active = (urls.indexOf(location.pathname) !== -1);
+                      if (active) {
```

Comment:
I'm not personally a huge fan of doing mutation within a `map`, maybe it would be clearer to change it to a `forEach` and append to a list at the end? Or even just extract this block into a function (if there's an easy way to do that)?